pysmt/cmd/install.py

```diff
@@ -547,46 +153,54 @@ def print_welcome():
 def main():
     options = parse_options()
```

Comment:
no arguments means install all the solvers in ```PYSMT_SOLVER``` env variable...

We could print a messgae like "Nothing to do. Try install.py --help" if no argument and ```PYSMT_SOLVER``` is empty.